prefork_test.go

```diff
@@ -71,7 +71,6 @@ func Test_App_Prefork_Master_Process(t *testing.T) {
 }
 
 func Test_App_Prefork_Child_Process_Never_Show_Startup_Message(t *testing.T) {
-	t.Parallel()
```

Comment:
This is most likely an unrelated change and should be reverted.

Reply:
Just try to fix data race on tests